lsp/source/server.mts

```diff
@@ -113,6 +115,7 @@ const diagnosticsDelay = 16;  // ms delay for primary updated file
 const diagnosticsPropagationDelay = 100;  // ms delay for other files
 
 connection.onInitialize(async (params: InitializeParams) => {
+  connection.console.log(`Initializing civet server at ${new Date().toLocaleTimeString()}`);
```

Comment:
All logs on the `server` should use the `console` on the connection as much as possible.